iftk/helpers/deepgram_stream_queue.py

```diff
@@ -0,0 +1,99 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import asyncio
+import json
+import pyaudio
+import websockets
+
+FORMAT = pyaudio.paInt16
+CHANNELS = 1
+RATE = 44100
+CHUNK = 512
+DEEPGRAM_CHUNK = 8000
+DEEPGRAM_RATE = 16000
+SECONDS = 2
+WSS_URL = "wss://api.deepgram.com/v1/listen?endpointing=500&encoding=linear16&sample_rate=16000&channels=1&interim_results=false"
+
+
+async def DeepgramStream(key: str, transcription_queue: asyncio.Queue) -> None:
+    """Put ASR results from Deepgram streaming API in an asyncio.Queue.
+
+    Args:
+        key (str): Your Deepgram API key
+        transcription_queue (asyncio.Queue): A queue that stores the transcriptions as a
+        "speech_final" message is detected from Deepgram.
+    """
+    extra_headers = {"Authorization": f"Token {key}"}
+    audio_queue = asyncio.Queue()
+
+    def callback(in_data, frame_count, time_info, status):
+        """A basic callback for Deepgram's stream."""
+        audio_queue.put_nowait(in_data)
+        return (None, pyaudio.paContinue)
+
+    async with websockets.connect(
+        WSS_URL,
+        extra_headers=extra_headers,
+    ) as ws:
+
+        async def keep_alive(websocket: websockets.WebSocketClientProtocol):
+            while True:
+                keep_alive_msg = json.dumps({"type": "KeepAlive"})
+                websocket.send(keep_alive_msg)
+                print("KeepAlive message sent")
```

Comment:
remove this debug print, or move it to a logger debug statement, e.g., 
```suggestion
                # above, after the imports (per https://docs.python.org/3/library/logging.html) : logger = logging.getLogger(__name__)
                logger.debug("KeepAlive message sent")
```